ceph-salt-formula/salt/ceph-salt/cephbootstrap.sls

```diff
@@ -38,9 +38,14 @@ run cephadm bootstrap:
                 --initial-dashboard-user {{ dashboard_username }} \
                 --output-keyring /etc/ceph/ceph.client.admin.keyring \
                 --output-config /etc/ceph/ceph.conf \
+                --skip-monitoring-stack \
```

Comment:
can you explain in the commit message why this line is needed (ideally, including an explanation of what the "monitoring stack" is)

Reply:
this particular change was moved to a dedicated PR - https://github.com/ceph/ceph-salt/pull/179